INFO:werkzeug: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
ERROR:app:Exception on /search [GET]
Traceback (most recent call last):
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\vin-i\PycharmProjects\Homeworks\HW_12\main\main.py", line 21, in page_tag
    posts = load_posts(POST_PATH)  # Загружаем списов всех постов
  File "C:\Users\vin-i\PycharmProjects\Homeworks\HW_12\functions.py", line 12, in load_posts
    return json.load(file)
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 293, in load
    return loads(fp.read(),
  File "C:\Users\vin-i\AppData\Local\Programs\Python\Python310\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xc0 in position 197: invalid start byte
INFO:werkzeug:127.0.0.1 - - [26/Mar/2022 22:20:03] "[35m[1mGET /search?s=блин HTTP/1.1[0m" 500 -
